src/choices.c

```diff
@@ -14,7 +14,7 @@
 #include "choices.h"
 
 size_t
-min_match_length(char *str, char *query)
+min_match_length(char *str, char *query, int *start_pos)
```

Comment:
Now that this methods also communicates back the start position of the match would renaming it `find_match`, accepting another pointer for the length and returning `1` if a match was found and `0` if not, make this clearer?